sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,7 +1,7 @@
-# 2019-09-24 1.1.0
+# 2019-09-26 1.1.0
 
 - Add browser support. Authentication using Azure Active Directory credentials is not supported yet - use a connection string instead.
-  [PR 5128](https://github.com/Azure/azure-sdk-for-js/pull/5128) related to [issue 3373](https://github.com/Azure/azure-sdk-for-js/issues/3373)
+  [PR 5253](https://github.com/Azure/azure-sdk-for-js/pull/5128) related to [issue 3373](https://github.com/Azure/azure-sdk-for-js/issues/3373)
```

Comment:
This change is not needed. PR 5128 was the one with the actual change that enables browser support by shipping the browser bundle as part of the package, so link to that is sufficient even if we plan to revert the version update to amqp-common

Reply:
Also the PR number in the link set doesnt match the actual link value

Reply:
Updated 

(for some weird reason I'm not getting GitHub notifications for your comments, but able to for others :/ did something change?)